invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -27,74 +27,80 @@
 {%- block page_body %}
   <section id="banners" class="banners" aria-label="{{ _('Information banner') }}">
 
-  <!-- COMMUNITY HEADER: hide it when displaying the submission request -->
-  {% if not is_preview_submission_request %}
-
-    {%- set parent = record.get("parent", {}) %}
-    {%- set community_review = parent.get("review", {}).get("receiver", {}).get("community") %}
-    {%- set community_default = parent.get("communities", {}).get("default") %}
-    {%- set has_community = community_review or community_default %}
-
-    {% if has_community %}
-      {%- set expanded_parent = record.get("expanded", {}).get("parent", {}) %}
-      {%- set community = expanded_parent.get("review", {}).get("receiver") or expanded_parent.get("communities", {}).get("default") %}
-      <div class="ui fluid container page-subheader-outer with-submenu compact">
-        <div class="ui container page-subheader">
-          <div class="page-subheader-element">
-            <img
-              class="ui mini image"
-              src="{{ community.links.logo }}"
-              onerror="this.onerror=null;this.src='/static/images/square-placeholder.png';"/>
-          </div>
-          <div class="page-subheader-element">
-              {{ community.metadata.title }}
+    <!-- COMMUNITY HEADER: hide it when displaying the submission request -->
+    {% if not is_preview_submission_request %}
+
+      {%- set parent = record.get("parent", {}) %}
+      {%- set community_review = parent.get("review", {}).get("receiver", {}).get("community") %}
+      {%- set community_default = parent.get("communities", {}).get("default") %}
+      {%- set has_community = community_review or community_default %}
+
+      {% if has_community %}
+        {%- set expanded_parent = record.get("expanded", {}).get("parent", {}) %}
+        {%- set community = expanded_parent.get("review", {}).get("receiver") or expanded_parent.get("communities", {}).get("default") %}
+        <div class="ui fluid container page-subheader-outer with-submenu compact">
+          <div class="ui container page-subheader">
+            <div class="page-subheader-element">
+              <img
+                class="ui mini image"
+                src="{{ community.links.logo }}"
+                onerror="this.onerror=null;this.src='/static/images/square-placeholder.png';"/>
+            </div>
+            <div class="page-subheader-element">
+              {% if community.visibility == "public" %}
+                <a
+                  href="{{ url_for("invenio_app_rdm_communities.communities_detail", pid_value=community.slug) }}"> {{ community.metadata.title }}</a>
+              {% else %}
+                {{ community.metadata.title }}
```

Comment:
why community header is not a link when the community is restricted?

Reply:
I changed it, i thought it would not make sense at first

Reply:
if the community can be seen from the user then there is no point in not being able to navigate to it!